modules/op-constructing-pipelines-using-existing-tasks.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-constructing-pipelines-using-existing-tasks_{context}"]
+= Constructing Pipelines using existing Tasks and ClusterTasks
```

Comment:
Does it make sense to rename it to - Constructing Pipelines using Pipeline Builder?

Reply:
Yeah makes sense, since it is the key point in this module